### DISCUSSION TITLE
EOM2 IOException

### DISCUSSION THREAD

**Discussion by MLipperts** [dcm4che/dcm4che#1233] [Q&A]
Hi all,

Not sure if this is best put under issues or discussion (since it's probably an issue, but not on the dcm4che side)
I'm trying to receive HL7 messages from an infusion system, and now have communication between the system and the hl7receiver. It does however throw an IOException (1CH followed by aH instead by 0DH). Can anyone help translating this exception into feedback I can send the vendor of the infusor?
Thanks in advance!

**Comment by gunterze** [comment]
quote [Transport Specification: MLLP, Release 1](https://www.hl7.org/documentcenter/public/wg/inm/mllp_transport_specification.PDF)
